Common.mk

```diff
@@ -307,6 +316,23 @@ $(GIT_PATCH_TARGET): $(GIT_CHECKOUT_TARGET)
 	$(BASE_DIRECTORY)/build/lib/go_mod_download.sh $(MAKE_ROOT) $(REPO) $(GIT_TAG) $(GOLANG_VERSION) $(REPO_SUBPATH)
 	@touch $@
 
+ifeq ($(HAS_HELM_CHART),true)
+$(HELM_REPOSITORY):
+	git clone $(CLONE_URL) $(HELM_REPOSITORY)
+
+$(HELM_GIT_CHECKOUT_TARGET): | $(HELM_REPOSITORY)
+	@echo rm -f $(HELM_REPOSITORY)/eks-anywhere-*
+	(cd $(HELM_REPOSITORY) && $(BASE_DIRECTORY)/build/lib/wait_for_tag.sh $(HELM_GIT_TAG))
+	git -C $(HELM_REPOSITORY) checkout -f $(HELM_GIT_TAG)
+	touch $@
+
+$(HELM_GIT_PATCH_TARGET): $(HELM_GIT_CHECKOUT_TARGET)
+	git -C $(HELM_REPOSITORY) config user.email prow@amazonaws.com
+	git -C $(HELM_REPOSITORY) config user.name "Prow Bot"
+	git -C $(HELM_REPOSITORY) am --committer-date-is-author-date $(or $(wildcard $(PROJECT_ROOT)/helm/patches),$(wildcard $(MAKE_ROOT)/patches))/*
```

Comment:
I think you want to change the last patches to helm/patches as well. These two wildcards are to cover patches folders in the root and in release branch folders, which ever may exists if either.

Reply:
Yeh, good call. I'd removed support for PROJECT_ROOT patches from the other place, so I'm going to pull it here. MAKE_ROOT helm/patches only at least for now